python/cm/job.py

```diff
@@ -953,14 +953,14 @@ def log_rotation():
             log.info('rotation log from db')
 
     if log_rotation_on_fs:
-        rotation_jobs_on_fs = JobLog.objects.filter(
-            finish_date__lt=timezone.now() - timedelta(days=log_rotation_on_fs)).values('id')
-
-        if rotation_jobs_on_fs:
-
-            for job in rotation_jobs_on_fs:
-                shutil.rmtree(os.path.join(config.RUN_DIR, str(job['id'])))
-            log.info('rotation log from fs')
+        for folder in os.listdir(config.RUN_DIR):
```

Comment:
Do you understand that it could be not only folders in RUN_DIR?

Reply:
And what else could be?

Reply:
File, regular file. You change logic, and create a lot of dirs in run, but on old installation there are a lot of files there.

Reply:
If I understand correctly, you want me to add the deletion of files from the old log storage location LOG_DIR?

Reply:
Added deletion of both folders and files